ci/archlinux.sh

```diff
@@ -22,6 +22,8 @@ pacman -Sy --noconfirm \
 	systemd-libs \
 	make \
 	pkgconf \
+	meson \
+	ninja \
```

Comment:
ninja should come as a dependency for most of the distros, doesn't it? So we do not need to install it explicitly.

Reply:
It doesn’t…

Reply:
Huh, it's an issue in arch then. Other distros hopefully have it right.

Reply:
I had to install it on all fedora and Debian aswell, pretty sure they aren’t on the base system. I’ll check again but I’m 99.99% sure

Reply:
At least on Debian, `meson` depend on `ninja-build`.

Reply:
Ok it seems to be ok removing them